repo/src/main/java/com/parashift/onlyoffice/Util.java

```diff
@@ -136,6 +136,26 @@ public String getEditorUrl() {
         return configManager.demoActive() ? configManager.getDemo("url") : (String) configManager.getOrDefault("url", "http://127.0.0.1/");
     }
 
+    public String getBackUrl(NodeRef nodeRef){
+        String path = "";
+        while(this.nodeService.getPrimaryParent(nodeRef).getParentRef() != null){
+            if(this.nodeService.getProperty(this.nodeService.getPrimaryParent(nodeRef).getParentRef(),
+                    ContentModel.PROP_NAME).toString().equals("Company Home")) {
```

Comment:
l10n?

Reply:
Company Home is the constant name of root folder. Moved to constant variable.